cuda/test/matrix/csr_kernels.cpp

```diff
@@ -86,8 +91,8 @@ class Csr : public ::testing::Test {
     void set_up_apply_data(std::shared_ptr<Mtx::strategy_type> strategy)
     {
         mtx = Mtx::create(ref, strategy);
-        mtx->copy_from(gen_mtx<Vec>(532, 231, 1));
-        expected = gen_mtx<Vec>(532, 1, 1);
+        mtx->copy_from(gen_mtx<Vec>(512, 231, 1));
+        expected = gen_mtx<Vec>(512, 1, 1);
```

Comment:
Why the change from 532 to 512?
532 seems like a good number to ensure that more than one thread block is used.